plugins/groups/src/components/MemberView/components/MemberSummary.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { Box, Grid, Link, Tooltip, Typography } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { InfoCard, Progress, useApi } from '@backstage/core';
+import { useAsync } from 'react-use';
+import { catalogApiRef } from '@backstage/plugin-catalog';
+import { UserEntity } from '@backstage/catalog-model';
+import Avatar from 'react-avatar';
```

Comment:
any benefit over this one? https://material-ui.com/components/avatars/

Reply:
Better fallback options but you might be right that is not worth to install additional package just for it. Gonna remove it.